Syncing docs across docker-compose and kubernetes for the common comp… (#47)
voxel51/fiftyone-teams-app-deploy: docker/README.md

## Initial Installation vs. Upgrades

> Only when performing an initial installation, in `compose.yaml` set

Reviewer: ```suggestion
When performing an initial installation, in `compose.yaml` set
```

`services.fiftyone-app.environment.FIFTYONE_DATABASE_ADMIN: true`.

> Otherwise, set `services.fiftyone-app.environment.FIFTYONE_DATABASE_ADMIN: false`.

Reviewer: ```suggestion
When performing a FiftyOne Teams upgrade, set `services.fiftyone-app.environment.FIFTYONE_DATABASE_ADMIN: false`.
```

See [Upgrade Process Recommendations](#upgrade-process-recommendations).

The environment variable `FIFTYONE_DATABASE_ADMIN` controls whether the database may be migrated.

> This is a safety check to prevent automatic database upgrades that will break other user's SDK connection.

Reviewer: ```suggestion
This is a safety check to prevent automatic database upgrades that will break other users' SDK connections.
```

      [./compose.plugins.yaml](./compose.plugins.yaml)
      instead of
      [./compose.yaml](./compose.yaml)

>     - Containers need the following access to the plugin storage

Reviewer: ```suggestion
    - Containers need the following access to plugin storage
```

      [./compose.dedicated-plugins.yaml](./compose.dedicated-plugins.yaml)
      instead of the
      [./compose.yaml](./compose.yaml)

>     - Containers need the following access to the plugin storage
>       - `teams-plugins` require `read`
>       - `fiftyone-api` require `read-write`

Reviewer: ```suggestion
    - Containers need the following access to plugin storage
      - `teams-plugins` requires `read`
      - `fiftyone-api` requires `read-write`
```
